tests/platform_tests/counterpoll/counterpoll_helper.py

```diff
@@ -1,9 +1,12 @@
 from tests.platform_tests.counterpoll.counterpoll_constants import CounterpollConstants
+from tests.common.devices.sonic_asic import SonicAsic
 
 
 class ConterpollHelper:
     @staticmethod
     def get_counterpoll_show_output(duthost):
+        if isinstance(duthost, SonicAsic):
```

Comment:
can we implement `show_and_parse` in `SonicAsic` we can avoid checking duthost is instance of `SonicAsic` or not

Reply:
done